src/js/tabs/motors.js

```diff
@@ -365,8 +365,28 @@ TABS.motors.initialize = function (callback) {
         // select current mixer configuration
         mixerListElement.val(FC.MIXER_CONFIG.mixer).change();
 
+        function validateMixerOutputs() {
+            MSP.promise(MSPCodes.MSP_MOTOR).then(() => {
+                const motors = mixerList[FC.MIXER_CONFIG.mixer - 1].motors;
+                self.numberOfValidOutputs = motors;
+                const msg = i18n.getMessage('motorsDialogMixerReset', { mixerMotors: motors });
+
+                for (let i = 0; i < FC.MOTOR_DATA.length; i++) {
+                    if (FC.MOTOR_DATA[i] === 0) {
+                        self.numberOfValidOutputs = i;
+                        if (i === 0 && motors > 0) {
```

Comment:
is that possible that i > 0 but still not equal to motors at this point?

Reply:
Yes, `i` should be less or equal to motors, if not there are not enough motor resources and the Mixer will be reset to default.

Reply:
Not sure we are checking the right thing here - shouldn't we be checking `i < motors` as well?